src/test/java/nl/stil4m/mollie/concepts/AbstractConceptTest.java

```diff
@@ -0,0 +1,63 @@
+package nl.stil4m.mollie.concepts;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class AbstractConceptTest {
+
+    @Test
+    public void testConstructor() {
+        assertEquals("https://api.mollie.com/v1",new ConceptStub("https://api.mollie.com/v1").url());
+        assertEquals("https://api.mollie.com/v1/api",new ConceptStub("https://api.mollie.com/v1","api").url());
+    }
+    
+    @Test(expected=NullPointerException.class)
```

Comment:
good point; hadn't really considered the API user when choosing the exception type